### PR TITLE
Fix typo in SetLowPowerDefault

### PR DESCRIPTION
**Pull request by ybakhano**

### CODE REVIEW

**Review by lakulako — DISMISSED** [review]

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp [hunk 1]
```diff
@@ -1412,12 +1412,12 @@ Return value:
...
-mfxStatus MfxHwH264Encode::SetLowPowerDefault(MfxVideoParam& par, const eMFXHWType& platfrom)
+mfxStatus MfxHwH264Encode::SetLowPowerDefault(MfxVideoParam& par, const eMFXHWType& platform)
 {
     mfxStatus sts = CheckTriStateOption(par.mfx.LowPower) == false ? MFX_WRN_INCOMPATIBLE_VIDEO_PARAM : MFX_ERR_NONE;
-    (void)platfrom; // fix wrn for non Gen11 build
-    if (  (platfrom == MFX_HW_JSL
-        || platfrom == MFX_HW_EHL
+    (void)platform; // fix wrn for non Gen11 build
+    if (  (platform == MFX_HW_JSL
+        || platform == MFX_HW_EHL
```
Comment (ivan-losev) [inline _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp:1420]: Please, guard below code at line 2265 into {}
```
            unsupported = true;
            par.mfx.RateControlMethod = 0;
```

**Review by ivan-losev — CHANGES_REQUESTED** [review]

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp [hunk 2]
```diff
@@ -1412,12 +1412,12 @@ Return value:
...
 MFX_ERR_NONE - if no errors
 */
-mfxStatus MfxHwH264Encode::SetLowPowerDefault(MfxVideoParam& par, const eMFXHWType& platfrom)
+mfxStatus MfxHwH264Encode::SetLowPowerDefault(MfxVideoParam& par, const eMFXHWType& platform)
 {
     mfxStatus sts = CheckTriStateOption(par.mfx.LowPower) == false ? MFX_WRN_INCOMPATIBLE_VIDEO_PARAM : MFX_ERR_NONE;
-    (void)platfrom; // fix wrn for non Gen11 build
-    if (  (platfrom == MFX_HW_JSL
-        || platfrom == MFX_HW_EHL
+    (void)platform; // fix wrn for non Gen11 build
```
Comment (lakulako) [inline _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp:1418]: Looks like (void)platform; is no more needed here.

**Review by lakulako — APPROVED** [review]

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp (hunk 1 above)
Comment (ybakhano) [reply _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp:1420]: done

File: _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp (hunk 2 above)
Comment (ybakhano) [reply _studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp:1418]: I suppose it can be done later

**Review by ivan-losev — APPROVED** [review]

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**